test/integration/patients/patient/sidebar.js

```diff
@@ -387,6 +402,16 @@ context('patient sidebar', function() {
     cy
       .get('.alert-box')
       .contains('Engagement settings for this patient could not be found.');
+
+    // TODO: Enable once Patient Form app is complete
```

Comment:
Still this right?

Reply:
Sorry @paulfalgout my gmail github filter apparently did not work. This should be ready to go.